libtsuba/include/tsuba/RDG.h

```diff
@@ -180,6 +180,16 @@ class KATANA_EXPORT RDG {
   /// Remove all edge properties
   void DropEdgeProperties();
 
+  //write the list of node and edge column names persisted to json, private as it is called only when the node and edge property index vectors are pushed back
+  void set_node_property_indexes_column_name(
+      std::vector<std::string>& node_property_indexes_column_name);
+  void set_edge_property_indexes_column_name(
+      std::vector<std::string>& edge_property_indexes_column_name);
+
+  // read the same as above and recreate indexes
+  std::vector<std::string>& get_node_property_indexes_column_name();
+  std::vector<std::string>& get_edge_property_indexes_column_name();
```

Comment:
Typically accessor functions  should be just the member name without the trailing `_`